stdlib/LinearAlgebra/src/symmetric.jl

```diff
@@ -254,11 +254,17 @@ end
     end
 end
 
+Base._reverse(A::Symmetric, dims::Integer) = reverse!(Matrix(A); dims)
+Base._reverse(A::Symmetric, dims::Colon) = Symmetric(reverse(A.data), A.uplo == 'U' ? :L : :U)
+
 @propagate_inbounds function setindex!(A::Symmetric, v, i::Integer, j::Integer)
     i == j || throw(ArgumentError("Cannot set a non-diagonal index in a symmetric matrix"))
     setindex!(A.data, v, i, j)
 end
 
+Base._reverse(A::Hermitian, dims) = reverse!(Matrix(A); dims)
+Base._reverse(A::Hermitian, dims::Colon) = Hermitian(reverse!(Matrix(A)))
```

Comment:
```suggestion
Base._reverse(A::Hermitian, dims::Colon) = Hermitian(reverse(A.data), A.uplo == 'U' ? :L : :U)
```
should be faster to avoid constructing `Matrix(A)`, similar to the method for `Symmetric`